jbt-it/neomdb
# #310 client workshop details admin

How to test:

- navigate to the workshop overview page
- select the workshop "Projektmanagement"
- check if the table is displayed correctly
- check if the button and dialog for editing the workshop works
- check if the button and dialog to add a new workshop instance works
- check if the button to delete the workshop words


closing #310  #311  #338 #339
